packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -753,8 +757,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private readonly dataStores: DataStores;
 
-    // This is the source of truth for writing in the summary metadata blob.
-    private readonly summaryGCFeature: Required<IContainerRuntimeMetadata>["gcFeature"];
+    // The current GC version that this container is running.
+    private readonly currentGCVersion = GCVersion;
```

Comment:
Do you really need this?

Reply:
Not really. I added this so I could update it in tests to simulate version updates. I can remove it if we can come up with other ways of testing :-).